### PR TITLE
Stream Analytics - 3rd party player

### PR DESCRIPTION
**Pull request by zaidf**

### CODE REVIEW

**Review by Everlag — APPROVED** [review]

**Review by YoutacVA — APPROVED** [review]

**Review by YoutacVA — APPROVED** [review]

File: products/stream/src/content/getting-analytics/fetching-bulk-analytics.md [hunk 1]
```diff
@@ -2,6 +2,12 @@
 pcx-content-type: reference
 ---
 
+ <Aside>
+
+Currently, Stream Analytics are only available for video plays that use the Stream Player. If you are using a 3rd party player, you will not see analytics for video plays from 3rd party players.
```
Comment (marciocloudflare) [inline products/stream/src/content/getting-analytics/fetching-bulk-analytics.md]: Change to `third-party player`

File: products/stream/src/content/getting-analytics/fetching-bulk-analytics.md [hunk 2]
```diff
@@ -2,6 +2,12 @@
...
+  
+    </Aside>
```
Comment (marciocloudflare) [inline products/stream/src/content/getting-analytics/fetching-bulk-analytics.md]: Remove extra spaces/tabs

**Comment by marciocloudflare** [comment]
Hey @zaidf. Could you check my comments? Thank you!

**Comment by KimJ15** [comment]
@sejoker would you be able to review and approve Márcio's comments? Thanks!

**Review by sejoker — APPROVED** [review]

**Comment by renandincer** [comment]
This PR broke navigation:
image

Aside need to be below the title

**Pull request merged**